netbox/ipam/forms/bulk_import.py

```diff
@@ -327,6 +327,13 @@ class IPAddressImportForm(NetBoxModelImportForm):
         to_field_name='name',
         help_text=_('Assigned interface')
     )
+    fhrpgroup = CSVModelChoiceField(
```

Comment:
Sorry, not sure how my previous comments got dropped, but they did.

Let's change this to `fhrp_group` (here and below) to maintain naming consistency.